lib/index.js

```diff
@@ -31,8 +31,20 @@ export default {
     this.subscriptions.add(atom.config.observe('flow-ide.onlyIfAppropriate', (onlyIfAppropriate) => {
       this.onlyIfAppropriate = onlyIfAppropriate
     }))
+
+    let priority = atom.config.get('flow-ide.hyperclickPriority')
```

Comment:
While the rest of the PR looks good and should work, `config.get` in activate and hyperclick provider is pretty expensive especially when we have an observe just a few lines below and will slow down Atom startup, couldn't you set this to null by default and update the check below, also we should store the value on `this` to avoid doing another get in `provideHyperclick`

```js
let restartNotification = null
this.hyperClickPriority = null
this.subscriptions.add(atom.config.observe('flow-ide.hyperClickPriority', (hyperClickPriority) => {
  if (this.hyperClickPriority !== null && restartNotification === null) {
    restartNotification = atom.notifications.addSuccess('Restart atom to update flow-ide priority?', {
      dismissable: true,
      buttons: [{
        text: 'Restart',
        onDidClick: () => atom.restartApplication(),
      }],
    })
    restartNotification.onDidDismiss(() => {
      restartNotification = null
    })
  }
  this.hyperclickPriority = hyperclickPriority
}))
```